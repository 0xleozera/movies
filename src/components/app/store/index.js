import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import reducers from '../reducers';

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const store = createStore(reducers, composeEnhancers(applyMiddleware(thunk, logger)));

export default store;