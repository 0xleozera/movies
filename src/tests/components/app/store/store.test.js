import store from '../../../../components/app/store';

test('[Redux] store should match snapshot', () => {
  const state = store.getState();
  expect(state).toMatchSnapshot();
});
