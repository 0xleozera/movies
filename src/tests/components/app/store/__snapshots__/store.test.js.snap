// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Redux] store should match snapshot 1`] = `
Object {
  "movies": Object {
    "movieDetails": Object {},
    "moviesFound": Array [],
  },
}
`;
